The definition of ACPI_VIDEO_CLASS was moved from video.c
to video.h in 3.1. Define it here to fix build for kernels older
than 3.1.

--- a/drivers/gpu/drm/i915/intel_opregion.c
+++ b/drivers/gpu/drm/i915/intel_opregion.c
@@ -307,6 +307,9 @@ static int intel_opregion_video_event(st
 	struct acpi_bus_event *event = data;
 	int ret = NOTIFY_OK;
 
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(3,1,0))
+#define ACPI_VIDEO_CLASS "video"
+#endif
 	if (strcmp(event->device_class, ACPI_VIDEO_CLASS) != 0)
 		return NOTIFY_DONE;
 
